release/assets/macos_cli_installation.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash -e
+
+USER_PROVIDED_PATH=$1
+
+main() {
+  cat << EOF
+   ______
+  / ____/___  ____    ( )_  _______
+ / /   / __ \/ __ \  / / / / / ___/
+/ /___/ /_/ / / / / / / /_/ / /
+\____/\____/_/ /_/_/ /\____/_/
```

Comment:
i would try separate between u and r - it is hard 4 me to read the r

Reply:
sure!